src/components/icons/LogoIcon.tsx

```diff
@@ -12,14 +13,16 @@ const LIGHT_GRADIENT_END = '#FFFFFF';
 const DARK_GRADIENT_START = '#22283B';
 const DARK_GRADIENT_END = '#555B6E';
 
-export const Logo: FC<ILogo> = ({
+export const LogoIcon: FC<ILogoIcon> = ({
   isDark,
   onClick,
-  className = '',
+  size = 'medium',
   ...props
-}: ILogo) => (
+}: ILogoIcon) => (
   <svg
-    className={className}
+    className={cn(
+      size === 'small' ? 'size-5' : size === 'medium' ? 'size-10' : 'size-16',
```

Comment:
I'm new to this clsx package 

What's your thoughts on

size === 'small' && 'size-5', size === 'medium' && 'size-10', size === 'large' && 'size-16'